### DISCUSSION TITLE
Dynamic price update

### DISCUSSION THREAD

**Discussion by williamwongys** [elimintz/justpy#413] [Q&A]
When I use rangeselector set to day:
```
    'rangeSelector': {
        'buttons': [{'type': 'hour', 'count': 1, 'text': '1h'},
                    {'type': 'hour', 'count': 4, 'text': '4h'},
                    {'type': 'day', 'count': 1, 'text': '1D'},
                    {'type': 'week', 'count': 1, 'text': '1W'},
                    {'type': 'month', 'count': 1, 'text': '1M'},
                    {'type': 'all', 'count': 1, 'text': 'All'}],
        'selected': 2,
        'inputEnabled': False},
```
The window stops showing the latest price, I have to manually drag to the right.  I saw the demo in highcharts website, it should automatic shows the latest price update.  Please help.

**Comment by WolfgangFahl** [comment]
@williamwongys  Thank you for your questions. I think it is a much better fit for https://stackoverflow.com/questions/tagged/justpy where you might potentially reach a bigger community. Also I am moving this to discussions because it's not clear whether it is an issue yet. You might follow the stackoverflow Q&A rules to find out.
